go-controller/pkg/cluster/cluster.go

```diff
@@ -26,12 +27,13 @@ func NewClusterController(kubeClient kubernetes.Interface, wf *factory.WatchFact
 	}
 }
 
-func setupOVNNode(nodeName string) error {
+func setupOVNNode(node *kapi.Node) error {
 	var err error
 
+	nodeName := node.GetName()
```

Comment:
@juanluisvaladas how about just using `node.Name` where we need it?

Reply:
Sure, I wonder why the function exists at all...